env/.env

```diff
@@ -3,4 +3,14 @@ STORAGE_URL=http://apbs.127.0.0.1.xip.io/storage
 ID_URL=http://apbs.127.0.0.1.xip.io/id/
 AUTOFILL_URL=http://apbs.127.0.0.1.xip.io/autofill
 VIZ_URL=http://apbs.127.0.0.1.xip.io/viz
-GA_TRACKING_ID=
\ No newline at end of file
+GA_TRACKING_ID=
+
+# React App-specific variables
+PROJECT_HOME_URL=http://poissonboltzmann.org/
+REGISTRATION_URL=http://eepurl.com/by4eQr
```

Comment:
Should this have a trailing "/"?

Reply:
A trailing slash isn't needed here, as the URL resolves successfully